tests/test_events.py

```diff
@@ -10,8 +10,18 @@ def test_post_events(
     auth_keys = [{"kid": "kid1", "secret": "secret1"},
                  {"kid": "kid2", "secret": "secret2"}]
     client = client_factory(auth_keys)
+    body = [{
```

Comment:
General comment: This may be premature, but I'm wondering if we shouldn't break tests out in a different manner so we don't end up with one long `test_events.py` (since everything here will end up being event-y). For example maybe:

```
test_auth.py
test_content_loaded.py
test_content_load_failed.py
```

The event specific test file can (if it happens) incorporate different versions. Also then  different "invalid body" tests can be written per event type as desired without confusion. We may want to rename some tests if we go this route (e.g. `test_post_events` is probably really `test_valid_jwt`).

Reply:
This sounds good to us. We'll update it.